syntax/entry.php

```diff
@@ -296,18 +298,23 @@ function replaceQuery() {
      * @param Doku_Renderer_plugin_data_edit $renderer
      */
     function _editData($data, &$renderer) {
-        $renderer->form->startFieldset($this->getLang('dataentry'));
-        $renderer->form->_content[count($renderer->form->_content) - 1]['class'] = 'plugin__data';
-        $renderer->form->addHidden('range', '0-0'); // Adora Belle bugfix
+        $renderer->form->addFieldsetOpen($this->getLang('dataentry'));
+        $renderer->form->getElementAt($renderer->form->elementCount() - 1)->attr('class', 'plugin__data');
+        $renderer->form->setHiddenField('range', '0-0'); // Adora Belle bugfix
```

Comment:
I think we can drop any support for Adora Belle

Reply:
Done.